hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/blockmanagement/TestReplicationPolicy.java

```diff
@@ -977,7 +977,6 @@ public void testChooseReplicaToDelete() throws Exception {
 
     //Even if this node has the most space, because the storage[5] has
     //the lowest it should be chosen in case of block delete.
-    storages[4].setRemainingForTests(100 * 1024 * 1024);
```

Comment:
This modification requires a more detailed explanation. The above comment indicates that line 980 is intended to construct a node with the most remaining space, so I think it should not be deleted.

Reply:
Hi, @zhangshuyan0 . Thanks a lot for reviewing.  In method TestReplicationPolicy#getDatanodeDescriptors, the datanodes[5] only has two storages: "storages[5]" and "storages[5]-extra". It does not have storages[4]. So, I think 
`storages[4].setRemainingForTests(100 * 1024 * 1024);` seems useless.  If we want to make datanodes[5]'s total remaining largest, we could set "storages[5]-extra".  What's your opinions?

Reply:
Thanks for your reply. Should we set the remaining of `storages[5]-extra` to `100 * 1024 * 1024`?